CMakeLists.txt

```
@@ -27,4 +27,4 @@ if(BUILD_TESTS)
     target_link_libraries(${TEST_NAME} gtest gtest_main)
```

Please, add this line and replace relative paths in tests includes:
```suggestion
    target_include_directories(${TEST_NAME} PUBLIC "source")
    target_link_libraries(${TEST_NAME} gtest gtest_main)
```

---

Done, thanks!